robotpy_ext/autonomous/selector.py

```diff
@@ -236,7 +236,11 @@ def run(self, control_loop_wait_time=0.020, iter_fn=None, on_exception=None):
             except:
                 on_exception()
             
-            iter_fn()
+            if type(iter_fn) in (list, tuple):
```

Comment:
…

please don't do this. use isinstance instead.

Reply:
does isinstance take a list?